src/backend/api/endpoints/userservice.py

```diff
@@ -27,9 +27,11 @@ async def current_user_and_player(request: Request) -> JSONResponse:
     mod_notifications = None
     if len(user_roles) > 0:
         mod_notifications = await handle(GetModNotificationsCommand(user_roles))
-    return JSONResponse(UserPlayer(user.id, user.player_id, user.email_confirmed, user.force_password_reset, player, user_roles, team_roles, series_roles, tournament_roles, mod_notifications))
+    tokens = await handle(GetUserAPITokensCommand(user.id))
```

Comment:
Can we put this inside the if len(user_roles) > 0 block as well for now? It's very rare for anyone to have tokens so I don't think it is worthwhile doing this on every request as this is done on every page load